pyodide_build/config.py

```diff
@@ -29,8 +30,15 @@ def __init__(self, pyodide_root: Path):
             **self._load_makefile_envs(),
             **self._load_config_file(Path.cwd(), os.environ),
             **self._load_config_from_env(os.environ),
+            **self._load_sysconfig_paths(),
         }
 
+    def _load_sysconfig_paths(self) -> Mapping[str, str]:
+        # Load the values of sysconfig.get_paths() with a
+        # "pyodide_sysconfig_" prefix to differentiate them
+        # from other configuration variables
+        return {f"pyodide_sysconfig_{k}": v for k, v in sysconfig.get_paths().items()}
```

Comment:
This would retrieve the host Python's sysconfig values, not those in the cross build env. We should parse these values from the `_sysconfigdata__emscripten_wasm32-emscripten.py` file in the xbuildenv.

Reply:
Where should I retrieve this file from the cross-build environment? I have noted that I can get the file when I do `pyodide xbuildenv install`, but if there's already a programmatic way to access this, I can use that instead of writing my own methods.

Reply:
Here is the code we locate the sysconfig file. Since it uses the variables in the Makefile.envs, you'll need to modify it a bit to use it in the config manager to prevent cyclic import.

https://github.com/pyodide/pyodide-build/blob/fac0109aa2acf14469320b049d710dd42639bf94/pyodide_build/pypabuild.py#L98-L102

I think you'll also need to read the [sysconfig.py](https://github.com/maxking/cpython-1/blob/master/Lib/sysconfig.py) and how it retrieves the value from the sysconfig file, as we cannot directly use `sysconfig` module here.